samples/index_tuning_sample/README.md

```diff
@@ -0,0 +1,272 @@
+
+This tutorial is hosted on [LangChain AlloyDB integration library](https://github.com/googleapis/langchain-google-alloydb-pg-python/tree/main/samples/index_tuning_guide) in Markdown format, accompanied by Python code sample files.
+
+
+## Before You Begin
+Make sure you have a Google Cloud project and billing is enabled.
+Install the gcloud CLI.
+Set gcloud project:
+
+gcloud config set project $PROJECT_ID
+Enable APIs:
+```bash
+gcloud services enable alloydb.googleapis.com \
+                       cloudresourcemanager.googleapis.com \
+                       compute.googleapis.com \
+```
+If you haven't already, create an AlloyDB database cluster and instance following this guide. 
+
+
+## Step 1: Clone sample code
+Run `git clone` command in your local directory to pull the sample code:
+git clone https://github.com/googleapis/langchain-google-alloydb-pg-python
+
+## Step 2: Move into your local sample directory:
+Move into the sample directory:
+cd samples/langchain-vector-search-index-optimization
+Step 3:  Install Dependencies
+In your local sample code directory, run this command to install required dependencies:
+pip3 install -r requirements.txt
+
+## Step 3: Fill In Database Info
+Enter your database information and credentials in `create_vector_embeddings.py`
+
+```python
+AlloyDB info
+PROJECT_ID = "duwenxin-space"
+REGION = "us-central1"  # @param {type:"string"}
+CLUSTER_NAME = "my-alloydb-cluster"  # @param {type:"string"}
+INSTANCE_NAME = "my-alloydb-instance"  # @param {type:"string"}
+DATABASE_NAME = "netflix"  # @param {type:"string"}
+USER = "postgres"  # @param {type:"string"}
+PASSWORD = "postgres"  # @param {type:"string"}
+```
+
+## Step 4: Generate Embeddings
+If you already have vector embeddings ready in your database, you can skip this step.
+Otherwise, create embeddings from the sample dataset `wine_reviews_dataset.csv` by running this command in your sample code directory:
+
+```bash
+python3 create_vector_embeddings.py
+```
+Now your database is populated with 100k vector embeddings.
+
+## Step 5 : Create Database Vector Store
+we create a database engine from the information you input previously, and create a vector store from that engine:
+
+```python
+from langchain_google_alloydb_pg import (
+    AlloyDBEngine,
+    AlloyDBVectorStore,
+)
+
+async def get_vector_store():
+    engine = await AlloyDBEngine.afrom_instance(
+        project_id=PROJECT_ID,
+        region=REGION,
+        cluster=CLUSTER_NAME,
+        instance=INSTANCE_NAME,
+        database=DATABASE_NAME,
+        user=USER,
+        password=PASSWORD,
+    )
+
+    vector_store = await AlloyDBVectorStore.create(
+        engine=engine,
+        distance_strategy=DISTANCE_STRATEGY,
```

Comment:
there is zero explanation of what `DISTANCE_STRATEGY` should be set to. If you are going to introduce a new variable you should a) explain the available options or b) default it

Reply:
added default